### PR TITLE
Fixes #8076: rudder agent status

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8076

### CODE REVIEW

File: share/lib/common.sh [hunk 1]
```diff
@@ -14,6 +14,39 @@ clear_colors() {
...
+  fi
+
+  if [ -n "${CMD}" ]
+  then
+    $CMD
+    RET="$?"
+    if [ "${action}" = "start" ] || [ "${action}" = "stop" ]; then
+      if [ $RET -eq 0 ]
+      then
+        [ "$QUIET" = false ] && printf "${GREEN}ok${NORMAL}: service ${service} has been ${action}ed"
```
Comment (amousset) [inline share/lib/common.sh]: Lacks a "\n" at the end.

File: share/lib/common.sh [hunk 2]
```diff
@@ -14,6 +14,39 @@ clear_colors() {
...
+      else
+        [ "$QUIET" = false ] && printf "${RED}error${NORMAL}: service ${service} could not be ${action}ed"
```
Comment (amousset) [inline share/lib/common.sh]: Same here

**Comment by peckpeck** [comment]
PR updated

**Comment by amousset** [comment]
Ok, merging.

**Pull request merged**